File: src/main/java/com/und/security/RestAuthenticationRequest.java
package com.und.security;

import java.io.Serializable;

/**
 * Created by shiv on 21/07/17.
 */
public class RestAuthenticationRequest implements Serializable {

    private static final long serialVersionUID = -8445943548965154778L;

    private String username;
    private String password;

    public RestAuthenticationRequest() {
        super();
    }

    public RestAuthenticationRequest(String username, String password) {
        this.setUsername(username);
        this.setPassword(password);
    }

    public String getUsername() {
        return this.username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return this.password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: src/main/kotlin/com/und/security/RestAuthenticationRequest.kt
package com.und.security

import java.io.Serializable

/**
 * Created by shiv on 21/07/17.
 */
class RestAuthenticationRequest : Serializable {
    var username: String? = null
    var password: String? = null

    constructor() : super() {}

    constructor(username: String, password: String) {
        this.username = username
        this.password = password
    }

    companion object {

        private const val serialVersionUID = -8445943548965154778L
    }
}
